src/test/java/redis/clients/jedis/commands/commandobjects/CommandObjectsGenericCommandsTest.java

```diff
@@ -1,23 +1,1034 @@
 package redis.clients.jedis.commands.commandobjects;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 import org.junit.Test;
 import redis.clients.jedis.Jedis;
 import redis.clients.jedis.RedisProtocol;
+import redis.clients.jedis.args.ExpiryOption;
+import redis.clients.jedis.params.RestoreParams;
+import redis.clients.jedis.params.ScanParams;
+import redis.clients.jedis.params.SortingParams;
+import redis.clients.jedis.resps.ScanResult;
+
+/**
+ * Tests related to <a href="https://redis.io/commands/?group=generic">Generic</a> commands.
+ */
+public class CommandObjectsGenericCommandsTest extends CommandObjectsStandaloneTestBase {
```

Comment:
CommandObjectsKeyCommandsTest

Reply:
I tried to preseve the categories from redis.io: https://redis.io/commands/?group=generic, hence the word "Generic". It felt like it would improve orientation, at least for newcomers to the code.

Reply:
Hmm, things are still changing :disappointed: